README.md

```diff
@@ -47,7 +47,9 @@ If your team wants to use this, follow [Getting started](#getting-started).
 
 Check out the the latest run of the
 [demo workflow](https://github.com/PaideiaDilemma/Tortillas/actions/workflows/demo_ci.yml?query=branch%3Amain),
-for examples of tortillas in action.
+for examples of tortillas in GitHub Actions.
+
+If you create a working GitLab CI workflow, we'd be happy if you could contribute it to Tortillas.
```

Comment:
Here I actually was trying to say "tortillas in action".
The new note below does not belong in the demo section I think.
Also I think I am still going to add our gitlab ci file in an adapted form as an example.

Reply:
I see, I adjusted it a bit. Feel free to change it again ;)